alacritty.yml

```diff
@@ -480,8 +480,8 @@
 # - `action`: Execute a predefined action
 #
 #   - ToggleViMode
-#   - Search
-#   - SearchReverse
+#   - SearchForwards
+#   - SearchBackwards
```

Comment:
I guess they should be without `s` on the end? Just `SearchForward` and `SearchBackward`.